committeeoversightapp/templates/delete.html

```diff
@@ -13,7 +13,7 @@ <h1>Delete Hearing Record</h1>
  <form method="POST" class="form">
    {% csrf_token %}
    <div class="float-right">
-     <a href="/list" class="btn btn-back">
+     <a href="{% url 'list-event' %}" class="btn btn-back">
```

Comment:
gotta love the `url` function. small but mighty refactor!

Reply:
You love to see it !